client_vpn/main.tf

```diff
@@ -58,11 +69,12 @@ resource "aws_ec2_client_vpn_authorization_rule" "this_internal_dns" {
   description            = "Authorization for ${var.endpoint_name} to DNS"
 }
 
+
 resource "aws_ec2_client_vpn_authorization_rule" "this_subnets" {
   for_each               = toset(var.subnet_cidr_blocks)
   client_vpn_endpoint_id = aws_ec2_client_vpn_endpoint.this.id
   target_network_cidr    = each.value
-  access_group_id        = var.access_group_id
```

Comment:
I ran into a boatload of problems here. I couldn't create conditional because of for-each, then I found a way to do it, but terraform cache broke everything. I'll take a look monday to see if I can get this going again.

Reply:
Figured it out!